data-es7.js

```diff
@@ -738,7 +763,10 @@ exports.tests = [
   category: 'strawman',
   link : 'https://github.com/DavidBruant/Map-Set.prototype.toJSON',
   exec: function(){/*
-    return JSON.stringify(new Map([['a', 'b'], ['c', 'd']])) === '[["a","b"],["c","d"]]';
+    var map = new Map();
+    map.set('a', 'b');
+    map.set('c', 'd');
+    return JSON.stringify(map) === '[["a","b"],["c","d"]]';
```

Comment:
Thank you, that sounds like exactly the right philosophy. Let me know if there's anything in this PR that needs adjustment, but it seems like it's good to go :-)